### DISCUSSION TITLE
Welcome to rpmbuild Discussions!

### DISCUSSION THREAD

**Discussion by naveenrajm7** [naveenrajm7/rpmbuild#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elisiumm** [comment]
Hello,

I would like use the Github Action to build a rpm package with the param -D el6 0 or 1.

In the main file this option is not set.

Do we have an other solution to do ? 

Example of Rpm Build command  : rpmbuild -D 'dist el5' -D 'el3 0' -D 'el4 0' -D 'el5 1' -D 'el6 0' -D 'el7 0' -bb ./SPECS/*.spec
